build.gradle

```diff
@@ -1,8 +1,6 @@
-import edu.wpi.first.gradlerio.deploy.roborio.RoboRIO
-
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2022.3.1"
+    id "edu.wpi.first.GradleRIO" version "2022.2.1"
```

Comment:
It doesn't seem like we should downgrade this

Reply:
we will upgrade that after we flash the roborios